packages/elements/src/react/sign-in/root.tsx

```diff
@@ -89,18 +97,21 @@ export type SignInRootProps = SignInFlowProviderProps & {
  */
 export function SignInRoot({
   children,
-  exampleMode,
+  exampleMode = false,
   fallback = null,
-  path = SIGN_IN_DEFAULT_BASE_PATH,
-  routing,
+  path: pathProp,
+  routing = ROUTING.path,
```

Comment:
Made the falsy `exampleMode` and `routing` props more explicit by setting the default values they'd be inferred to anyways